library/src/main/java/me/toptas/fancyshowcase/FancyShowCaseView.java

```diff
@@ -363,6 +383,9 @@ public boolean onPreDraw() {
                         int startRadius = 0;
                         if (mView != null) {
                             startRadius = mView.getWidth() / 2;
+                        } else {
+                            mCenterX = mFocusPositionX;
```

Comment:
If there is no view to focus, reveal animation center should be center the of the device. These 2 lines overrides `mCenterX` and `mCenterY`. Please see lines 224, 225

You can test it in sample app. Click `NOFOCUS` button. Circular reveal should start from center of device.